<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1026</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>1011</width>
      <height>631</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>ordonnance</string>
     </attribute>
     <widget class="QTabWidget" name="rechercher">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>951</width>
        <height>521</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Ajouter une ordonnance</string>
       </attribute>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>480</x>
          <y>60</y>
          <width>47</width>
          <height>16</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(85, 0, 255);
font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>prenom</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(85, 0, 255);
font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>numordonnance</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_ajouter">
        <property name="geometry">
         <rect>
          <x>610</x>
          <y>410</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>700</x>
          <y>410</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>modifier</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>200</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(85, 0, 255);
font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>medicamen1</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>250</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(85, 0, 255);
font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>medicament2</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>295</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(85, 0, 255);
font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>medicamnet3</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>60</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(85, 0, 255);
font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>nom</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>690</x>
          <y>20</y>
          <width>71</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(85, 0, 255);
font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>nomdocteur</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>14</x>
          <y>350</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(85, 0, 255);
font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>medicament4</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="num">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>10</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="doc">
        <property name="geometry">
         <rect>
          <x>780</x>
          <y>20</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="nom">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>60</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="prenom">
        <property name="geometry">
         <rect>
          <x>550</x>
          <y>60</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="med1">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>200</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="med2">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>250</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="med3">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>310</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="med4">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>350</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="picture">
        <property name="geometry">
         <rect>
          <x>4</x>
          <y>5</y>
          <width>931</width>
          <height>501</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>60</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(85, 0, 255);
font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>nump</string>
        </property>
       </widget>
       <zorder>picture</zorder>
       <zorder>label_2</zorder>
       <zorder>label_3</zorder>
       <zorder>pb_ajouter</zorder>
       <zorder>pushButton</zorder>
       <zorder>label_6</zorder>
       <zorder>label_7</zorder>
       <zorder>label_8</zorder>
       <zorder>label</zorder>
       <zorder>label_9</zorder>
       <zorder>label_10</zorder>
       <zorder>num</zorder>
       <zorder>doc</zorder>
       <zorder>nom</zorder>
       <zorder>prenom</zorder>
       <zorder>med1</zorder>
       <zorder>med2</zorder>
       <zorder>med3</zorder>
       <zorder>med4</zorder>
       <zorder>lineEdit</zorder>
       <zorder>label_11</zorder>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Afficher les ordonnance</string>
       </attribute>
       <widget class="QTableView" name="tabordo">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>110</y>
          <width>931</width>
          <height>311</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="numm">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>30</y>
          <width>221</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>40</y>
          <width>61</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(85, 0, 255);
font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>numordonnance</string>
        </property>
       </widget>
       <widget class="QLabel" name="picture_2">
        <property name="geometry">
         <rect>
          <x>4</x>
          <y>5</y>
          <width>941</width>
          <height>491</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="pdf">
        <property name="geometry">
         <rect>
          <x>750</x>
          <y>440</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>imprimer</string>
        </property>
       </widget>
       <zorder>picture_2</zorder>
       <zorder>tabordo</zorder>
       <zorder>numm</zorder>
       <zorder>label_5</zorder>
       <zorder>pdf</zorder>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Supprimer une ordonnance</string>
       </attribute>
       <widget class="QLineEdit" name="num_2">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>30</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(85, 0, 255);
font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>numordo</string>
        </property>
       </widget>
       <widget class="QPushButton" name="supprimer">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>140</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
       <widget class="QLabel" name="picture_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>5</y>
          <width>941</width>
          <height>491</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <zorder>picture_3</zorder>
       <zorder>num_2</zorder>
       <zorder>label_4</zorder>
       <zorder>supprimer</zorder>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string/>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1026</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
